browser/src/slideshow/VideoRenderer.ts

```diff
@@ -88,20 +95,30 @@ abstract class VideoRenderer {
 		this.videoRenderInfo.deleteResources(this._context);
 	}
 
+	public handleClick(): void {
+		if (this.videoRenderInfo.playing) {
+			this.pauseVideo();
+		} else if (this.videoRenderInfo.ended) {
+			this.playVideo(true);
```

Comment:
don't we need deleteResources before playing again?

Reply:
I discussed this with Szymon, and it looks like what he was seeing was a memory leak which is fixed here: https://github.com/CollaboraOnline/online/pull/11726

therefore, we don't need that and I'm going to merge this PR :)